### PR TITLE
Add GPU direct test using host device

### PR DESCRIPTION
**Pull request by abdallahyas**
Added a test to test GPU direct traffic using host device for the
secondary network.

### CODE REVIEW

**Review by Mmduh-483 — APPROVED** [review]

File: nic_operator/nic_operator_ci_test.sh [hunk 1]
```diff
@@ -283,6 +283,65 @@ function test_ofed_and_rdma {
...
+    sleep 10
+
+    test_ofed_drivers
+    let status=status+$?
+    if [ "$status" != 0 ]; then
+        echo "Error: Ofed modules failed!"
+        return $status
+    fi
+
+    test_gpu_direct "harbor.mellanox.com/cloud-orchestration/cuda-perftest:Ubuntu20.04-cuda-devel-11.2.1"\
```
Comment (e0ne) [inline nic_operator/nic_operator_ci_test.sh:322]: I'm wondering if we want to publish this image to some public registry

Comment (adrianchiris) [reply nic_operator/nic_operator_ci_test.sh:322]: we got this project with example Dockerfiles: https://github.com/Mellanox/k8s-images/tree/master/perftest
however as this is used primarily for testing there was no request to have a CI/CD pipe to publish pre-build images.

the cuda base image is available in dockerhub where users can use to build their cuda containers.

Comment (e0ne) [reply nic_operator/nic_operator_ci_test.sh:322]: Fair enough. LGTM

**Review by e0ne — APPROVED** [review]

**Pull request merged**